app/controllers/vendor_api/decisions_controller.rb

```diff
@@ -123,11 +123,5 @@ def offer_service_for(application_choice, course_option)
         MakeOffer.new(offer_params(application_choice, course_option))
       end
     end
-
-    def validate_course_is_in_current_recruitment_cycle!
```

Comment:
Im wondering why we moved the validation out of the decision controller into the query service? The query service although only used here can be used possibly elsewhere and doesn't care about the recruitment year being the same? We only care about this logic whilst making/changing an offer?